### PR TITLE
Tweaks & improvements

### PR DESCRIPTION
**Pull request by aristath**
## Context
I was doing a code review today of the plugin, so this PR is just fixing & improving some things I came across.

## Summary

This PR can be summarized in the following changelog entry:

* Fix i18n issues for strings contained in JS files
* Minor code cleanup & readability tweaks

## Quality assurance

* [x] I have tested this code to the best of my abilities.
* [x] I have checked that the base branch is correctly set.

Fixes #

### CODE REVIEW

File: assets/js/todo.js
Comment (aristath) [inline assets/js/todo.js:1]: Fixed i18n issues in this file

File: classes/activities/class-content.php [hunk 1]
```diff
@@ -88,13 +88,16 @@ public function get_points_on_publish_date() {
...
-			$points *= Base::$points_config['content']['word-multipliers'][350];
-		} elseif ( $words > 100 ) {
-			$points *= Base::$points_config['content']['word-multipliers'][100];
+			return (int) ( $points * $multipliers[1000] );
+		}
+		if ( $words > 350 ) {
+			return (int) ( $points * $multipliers[350] );
+		}
+		if ( $words > 100 ) {
+			return (int) ( $points * $multipliers[100] );
```
Comment (jdevalk) [inline classes/activities/class-content.php:100]: Should this be a `switch`? Think that's easier to read.

Comment (aristath) [reply classes/activities/class-content.php:100]: We can't use `>` comparisons with a `switch`...

Comment (jdevalk) [reply classes/activities/class-content.php:100]: Hmm true, although we could do:

```php
switch (true) {
    case $words > 1000:
        do stuff;
        break;

    case $words > 350:
        do stuff;
        break;

    case $words > 100:
        do stuff;
        break;

    default:
        do stuff;
        break;
}
```

But doesn't add that much either.

**Review by jdevalk — APPROVED** [review]

**Pull request merged**